README.md

```diff
@@ -120,6 +120,27 @@ No native dependencies.
 
 Build with `cargo` and use `ldd` to check that the resulting binary does not depend on OpenSSL anymore.
 
+#### Linux and Docker
+
+Also you can build the repo with 
+[Parity CI Docker image](https://github.com/paritytech/scripts/tree/master/dockerfiles/cachepot-ci):
+
+```bash
+docker pull paritytech/cachepot-ci:staging
+mkdir ~/cache
+sudo chown 1000:1000 ~/cache #processes in the container runs as "nonroot" user with UID 1000
```

Comment:
1000 is a bold assumption, `${USER}:${USER}` should be an improvement.

Reply:
1000 is not an assumption - this is user id in the `docker.io/paritytech/cachepot-ci:staging` container. So you have to manipulate with this ID and permissions if it differs from user ID.

Reply:
Meta culpa.

In that case we should add the appropriate `--userns=..` option and the :z modifier for selinux to the mapping